src/modules/scoreboard.js

```diff
@@ -0,0 +1,18 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Hs3yRgIp4wIYr/scores';
+const leaderboard = document.querySelector('.list-score');
+
+export const display = (scores) => {
+  leaderboard.innerHTML = '';
+  if (scores.length === 0) {
+    leaderboard.innerHTML = '<li>No Scores to be displayed</li>';
+  }
+  scores.sort((a, b) => b.score - a.score).forEach((score) => {
+    leaderboard.innerHTML += ` <li class="Scores">${score.user}  ${score.score}</li>`;
+  });
+};
+
+export const fetchData = async () => {
+  await fetch(url)
+    .then((response) => response.json())
+    .then((json) => display(json.result));
+};
```

Comment:
I noticed you did not stick to the frame work design of this app, it says in the requirements that you have to stick to the wireframe design and you can add styling of you choice

This is your design:

![leader snip](https://user-images.githubusercontent.com/79506105/211170743-043282ca-78a9-4b4f-9039-153e64e74593.PNG)

Please stick to this design:
![framework snip](https://user-images.githubusercontent.com/79506105/211170755-6f14e33a-81df-44ff-b384-20b9e7442a83.PNG)